scheduler/docs/configuration.asc

```diff
@@ -98,6 +98,13 @@ We'll look at the configurable options in turn:
   This sets the role that Cook will connect to Mesos with. Default: `*`
   You can omit this property unless you've enabled security features in Mesos. The value should be in authorized list for the current `:principal` in `register_frameworks` Action in Mesos Authorization file. See http://mesos.apache.org/documentation/latest/authorization/ for details.
 
+`:enable-gpu-support`::
```

Comment:
That is fine. I meant put a comment somewhere (probably here, maybe somewhere else is better) that references Kevin's doc and anything else (sounds like not) needed to get mesos+cook in a position where it will schedule gpus